tests/ZiraLink.UnitTests/Services/ProjectServiceTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+
+using Moq;
+
+using ZiraLink.Api.Application;
+using ZiraLink.Api.Application.Services;
+using ZiraLink.Api.Application.Tools;
+using ZiraLink.Domain.Enums;
+using ZiraLink.UnitTests.Tools;
+namespace ZiraLink.UnitTests.Services
+{
+    public class ProjectServiceTests
+    {
+        private readonly ProjectService? _projectService;
```

Comment:
No need to make the variables nullable here, they are never null.

Reply:
I agree. Done.